io.playnux.calculator.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+app-id: io.playnux.calculator
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: io.playnux.calculator
+rename-desktop-file: calculator.desktop
+
+finish-args:
+  - '--device=dri'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+modules:
+  - name: calculator
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/PlayNux/calculator.git
+        branch: master
+        commit: e4d60fb4624bf80a49059e8f9de4d8edc2374307
+    post-install:
+      - install Dm644 io.playnux.calculator.appdata.xml -t ${FLATPAK_DEST}/share/metainfo/
+      
+  - name: granite
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/elementary/granite
+        branch: master
```

Comment:
you need a specific revision here too.

Also since it's a depenency it shoul probably be built before the application.